.gitignore

```diff
@@ -29,3 +29,6 @@ utilities/rdbmigration/.work
 **/.classpath
 **/.project
 **/bin/
+/example-scripts/bash-scripts/full-harvest-examples/1.13-1.15-examples/example-openalex/logs/
+/example-scripts/bash-scripts/full-harvest-examples/1.13-1.15-examples/example-openalex/data/
+/example-scripts/bash-scripts/full-harvest-examples/1.13-1.15-examples/example-openalex/previous-harvest/
```

Comment:
I was thinking that lines 3-7 includes this case?

Reply:
I think that adding:
```
**/data
**/logs
**/previous-harvest
```
to root-level `.gitignore` will solve this issue.